tests/core/test_block_manager.py

```diff
@@ -274,3 +274,48 @@ def test_reset():
     # Resetting block manager frees all allocated blocks.
     block_manager.reset()
     assert block_manager.get_num_free_gpu_blocks() == original_blocks
+
+
+def test_sliding_window_multi_seq():
```

Comment:
Can we add a small docstring explaining what runtime property this test verifies and the basic methodology?